src/globus_sdk/services/compute/client.py

```diff
@@ -71,3 +71,66 @@ def delete_function(self, function_id: UUIDLike) -> GlobusHTTPResponse:
                     :ref: Functions/operation/delete_function_v2_functions__function_uuid__delete
         """  # noqa: E501
         return self.delete(f"/v2/functions/{function_id}")
+
+    def get_task(self, task_id: UUIDLike) -> GlobusHTTPResponse:
+        """Get information about a task.
+
+        :param task_id: The ID of the task.
+
+        .. tab-set::
+
+            .. tab-item:: API Info
+
+                .. extdoclink:: Get Task
+                    :service: compute
+                    :ref: Tasks/operation/get_task_status_and_result_v2_tasks__task_uuid__get
+        """  # noqa: E501
+        return self.get(f"/v2/tasks/{task_id}")
+
+    def get_task_batch(self, task_ids: list[UUIDLike]) -> GlobusHTTPResponse:
```

Comment:
`list[T]` is problematic in public interfaces because it's invariant, meaning that this won't accept values of type `list[str]` or `list[UUID]`.
Typically, we accept `Sequence[T]` or `Iterable[T]` in such cases.

A good example would be here in the Auth client:
https://github.com/globus/globus-sdk-python/blob/b571dd28680eff8a964f4ee9e02e82296bba6658/src/globus_sdk/services/auth/client/service_client.py#L577

One thing you need to watch out for is that `str` is a `Sequence[str]`, so if we update the type to `t.Iterable[UUIDLike]`, you'll find that `client.get_task_batch(self, "foo")` is accepted by the type checker.

To deal with this, we have several cases which accept singular values in some APIs which require lists, and then explicitly wrap the content into a list. The whole situation represents a slight impedance mismatch between API contracts and Python typing constructs, so we need to flex things a little bit in one way or another to get clean types and runtime-safe code. For the example method from above, the handling code is here:
https://github.com/globus/globus-sdk-python/blob/b571dd28680eff8a964f4ee9e02e82296bba6658/src/globus_sdk/services/auth/client/service_client.py#L628

I recommend the same approach here, so that the signature becomes
```python
    def get_task_batch(self, task_ids: UUIDLike | t.Iterable[UUIDLike]) -> GlobusHTTPResponse:
```
and we use the "safe iter" helper from the `utils` module to "fix it in post", as it were.

Reply:
Oh, also, if you spot any existing SDK code using `list[T]` where `T` is a union type, in a public interface, that's probably a typing bug. If you can point those out to us, I'll work on fixes!

Reply:
Good call; I just pushed some updates.